src/plugins/d3/renderer/hooks/useShapes/HtmlLayer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {Portal} from '@gravity-ui/uikit';
+
+import {HtmlItem} from '../../types';
+
+type Props = {
+    htmlLayout: HTMLElement | null;
+    items: HtmlItem[];
+};
+
+export const HtmlLayer = (props: Props) => {
+    const {items, htmlLayout} = props;
+
+    if (!htmlLayout) {
+        return null;
+    }
+
+    return (
+        <Portal container={htmlLayout}>
+            {items.map((item, index) => {
+                return (
+                    <div
+                        key={index}
```

Comment:
I suppose we should use some uniq ids here

Reply:
There doesn't seem to be anything unique inside the html Item. But maybe it's not necessary here? it is used only for displaying elements - there is no internal state or any events inside